ui/src/timeMachine/components/view_options/TableOptions.tsx

```diff
@@ -131,6 +132,9 @@ export class TableOptions extends Component<Props, {}> {
             onSetThresholds={onSetColors}
           />
         </Grid.Column>
+        <Grid.Column>
```

Comment:
adds a toggle to allow users to switch between threshold colors being set to the background or text color